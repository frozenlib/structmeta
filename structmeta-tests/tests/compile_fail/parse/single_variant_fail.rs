structmeta_tests::parse_single_variant!(10 10);

fn main() {}
