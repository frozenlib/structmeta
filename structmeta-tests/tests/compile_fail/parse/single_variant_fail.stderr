error: parse failed.
 --> $DIR/single_variant_fail.rs:1:41
  |
1 | structmeta_tests::parse_single_variant!(10 10);
  |                                         ^^
